.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+[Notion link](notion-link-here) <!-- Remove this link if no relevant Notion link -->
+
+This PR adds... <!-- A brief description of what your PR does -->
+
+<!-- This checklist is just a guideline.-->
+<!-- If any of the following does not apply to your PR, please leave them unchecked and explain in the PR description -->
+
+- [ ] I've kept this PR as small as possible (~600 lines) by splitting it into PRs with manageable chunks of code
+- [ ] I've requested reviews from 1-2 reviewers
+- [ ] I've added [accessibility IDs](https://www.notion.so/dc5f4ce910f7431c84344ac79344e9f5?v=d487366816834dd4ab8dc12e0b5928f3) to my components
+- [ ] I've added/updated unit/integration tests
+- [ ] I've added/updated [e2e tests](https://www.notion.so/e2e-Documentation-a096d3e0bf75485e85ad692af8371ef3)
```

Comment:
hmm, do you think we should specify that it's the e2e tests for just the happy flow? :O would that be clearer haha

Reply:
Added, wrote "at least the happy flow" since it's OK to also write for the unhappy flow

Reply:
nice, thanks!! :D